.github/workflows/deploy.yml

```diff
@@ -0,0 +1,43 @@
+name: Deploy main branch
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - "cds-snc-docs/**"
+
+jobs:
+  deploy:
+    name: deploy
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
```

Comment:
Are you planning to add more Node versions to test?  If no, we could get rid of the `strategy` block and just have a global environment variable that set the Node version to use.

```
env:
   NODE_VERSION: 14.x
   
- name: Use Node.js ${{ env.NODE_VERSION }}
   uses: actions/setup-node@v1
   with:
     node-version: ${{ env.NODE_VERSION }}